shishito/runtime/platform/shishito_control_test.py

```diff
@@ -80,3 +80,6 @@ def test_init(self, url):
 
         :param str url:
         """
+        self.driver.get(url)
+        self.driver.implicitly_wait(int(self.shishito_support.get_opt('default_implicit_wait')))
+
```

Comment:
Actually we have control_test file for mobile execution, but it's empty and if I put it there I need also define case if user run mobile app using appium instead of browser, or just browser on mobile, I guess this can be removed from the main file for now